runtime/ops/fs.rs

```diff
@@ -246,8 +246,11 @@ async fn op_seek_async(
     .borrow_mut()
     .await;
 
-  let pos = (*fs_file).0.as_mut().unwrap().seek(seek_from).await?;
-  Ok(pos)
+  let std_file = fs_file.0.as_mut().unwrap();
+  let std_file: &'static mut StdFile = unsafe { std::mem::transmute(std_file) };
```

Comment:
This is unnecessarily unsafe - use a RefCell or some other type safe way of achieving this.